dashboard/test/dsl/dsl_test.rb

```diff
@@ -620,7 +632,6 @@ def setup
   end
 
   test 'serialize new_name, family_name, version_year and is_stable' do
-    puts 'test_serialize_new_name_and_family_name'
```

Comment:
Did you mean to delete this?

Reply:
yes. it looks like debug code accidentally included in a previous PR. since i was the one who added this test, I wasn't too worried anyone would mind it going away.